signer/src/network/libp2p/swarm.rs

```diff
@@ -322,30 +330,19 @@ impl SignerSwarm {
         let local_peer_id = *self.swarm.lock().await.local_peer_id();
         tracing::info!(%local_peer_id, "starting signer swarm");
 
-        // Start listening on the listen addresses.
-        for addr in self.listen_addrs.iter() {
-            self.swarm
-                .lock()
-                .await
-                .listen_on(addr.clone())
-                .map_err(|e| SignerSwarmError::LibP2P(Box::new(e)))?;
-        }
+        {
+            let mut swarm = self.swarm.lock().await;
```

Comment:
We just locked, and unlock the swarm a moment ago for the `local_peer_id`. Maybe we should move that log down here.

Reply:
[ffff750](https://github.com/stacks-network/sbtc/pull/1257/commits/ffff750d9634716e2f46315d6c1e56ca8440e0b0)